test/Microsoft.OpenApi.Tests/Expressions/PathExpressionTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.OpenApi.Expressions;
+using Xunit;
+
+namespace Microsoft.OpenApi.Tests.Writers
+{
+    [Collection("DefaultSettings")]
+    public class QueryExpressionTests
```

Comment:
>QueryExpressionTests [](start = 17, length = 20)

QueryExpressionTests and PathExpressionTests file names are swapped

Reply:
Oh, yes. Swapped.